bootstrap/kubeadm/api/v1beta1/kubeadmconfig_webhook.go

```diff
@@ -155,6 +157,49 @@ func (c *KubeadmConfigSpec) validateFiles(pathPrefix *field.Path) field.ErrorLis
 	return allErrs
 }
 
+func (c *KubeadmConfigSpec) validateUsers(pathPrefix *field.Path) field.ErrorList {
+	var allErrs field.ErrorList
+
+	for i := range c.Users {
+		user := c.Users[i]
+		if *user.Passwd != "" && user.PasswdFrom != nil {
```

Comment:
should this be 
```suggestion
		if user.Passwd !=  nil && user.PasswdFrom != nil {
```
otherwise it could panic...